test/remote/gateways/remote_aafes_test.rb

```diff
@@ -0,0 +1,148 @@
+require 'test_helper'
+
+class RemoteAafesTest < Test::Unit::TestCase
+  def setup
+    @gateway = AafesGateway.new(fixtures(:aafes))
+    @gateway.ssl_strict = false # Fails to send request to gateway otherwise :shrug
+    @amount = '%.2f' % 100
```

Comment:
This seems like an odd workaround. I would expect this logic to be included in the gateway code, not within the remote test code.